tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -50,9 +50,12 @@ def send_message(
         self,
         monitor_slug: str,
         guid: Optional[str] = None,
+        ts: Optional[datetime] = None,
         **overrides: Any,
     ) -> None:
-        now = datetime.now()
+        if ts is None:
+            ts = datetime.now()
```

Comment:
just make this the default instead of `None`

Reply:
That won't work. The default parameters of a function are evaluated at module load time. In this case _maybe_ that's okay since it's a test, but I think it's best not to get into that habit.